### ISSUE TITLE
Undeclared dependency on ftw.subsite

### ISSUE THREAD

**Issue by djowett** [OneGov/plonetheme.onegovbear#145]
I was just trying this out (so I could try out ftw.simplelayout) and I get

     ZopeXMLConfigurationError: File "/home/daniel/buildout-cache-UCS2/eggs/plonetheme.onegovbear-1.3.1-py2.7.egg/plonetheme/onegovbear/viewlets/configure.zcml", line 5.2-11.6
    ImportError: No module named subsite.interfaces

I'm guessing this code is wrong since you have a PR #37 to drop ftw.subsite dependency a while back.

**Comment by maethu** [comment]
awww... I'm gonna check this!

**Comment by maethu** [comment]
@djowett btw: you may try the latest version of ftw.simplelayout. It ships along with a suitable Theme. The plonetheme.blueberry. On Plone install time, also select the theme and ftw.simplelayout contenttypes.

**Comment by djowett** [comment]
Thanks @maethu. I already found the plonetheme.blueberry mentioned in the development.cfg of ftw.simplelayout.  Perhaps you should update the [README](https://github.com/4teamwork/ftw.simplelayout#alpha-state) to direct people to that rather than plonetheme.onegovbear?

btw (and getting seriously off-topic) It would be good to get your thoughts in [this community post](https://community.plone.org/t/high-expectations-of-site-authoring-and-page-layout/3087/15). 
I'm tempted in theory to use ftw.simplelayout at some point - but not ready to buy in to ftw.theming yet (with SCSS etc).   I'd be interested to know if the datamodel (blocks being stored as children of content IIUC) is working well? by comparison collective.cover uses annotations (and regrets it), mosaic uses plone.app.tiles and plone.app.blocks somehow and seems fairly robust at that level.

**Comment by maethu** [comment]
@djowett 
I made a issue for my colleague (https://github.com/4teamwork/ftw.simplelayout/issues/424) He will fix the readme.

ftw.simplelayout is extremely css heavy. 
First we tried to be as Theme independent as possible, but a aprox. a year ago we dropped this , because it was to hard to make proper styles without any side-effects. 

If you don't use the old sunbirst theme it may run without the blueberry theme. Just with `ftw.theming (base)`. `ftw.theming`is need for scss cooking/integration.
